gigascience/gigadb-website
# fix: add missing live env build and deploy jobs for spreadsheet uploader

# Pull request for issue: #<insert relevant github issue numbers that this PR is related to>

This is a pull request for the following functionalities:

Fix the issue whereby the spreadsheet upload tool couldn't be deployed on live environment

## How to test?

1. Deploy a complete staging environmment on your fork
2. Deploy a complete live environment on your fork
3. run the gitlab pipeline on this branch

Eventually, you should see the following jobs running and passing: 

* XLSUploaderBuildLive
* XLSUploaderDeployLive

## How have functionalities been implemented?

in the `excel-spreadsheet-uploader` project directory, there was a file `gitlab-config.yml` that defined the build and deploy jobs for staging environment. I've renamed that file `gitlab-config-staging.yml` and created a similar one called `gitlab-config-live.yml` that defines the build and deploy jobs for the live environment. Both files are then included in the main `.gitlab-ci.yml` config file.
